src/anomalous-value/anomalous-value.js

```diff
@@ -13,40 +13,35 @@ const { anomalousValue, aaveEverestId: AAVE_EVEREST_ID } = require('../../agent-
 const iface = new ethers.utils.Interface(abi);
 
 // events we are interested in
-const eventNames = ['Borrow', 'Deposit', 'Repay', 'Withdraw'];
+const eventFragments = [
+  iface.getEvent('Borrow'),
+  iface.getEvent('Deposit'),
+  iface.getEvent('Repay'),
+  iface.getEvent('Withdraw'),
+];
 
 // create rolling math object structure
 const rollingEventData = {};
 
 // helper function to create alerts
-function createAlert(log) {
+function createAlert(log, txEvent) {
   return Finding.fromObject({
     name: `High AAVE ${log.name} Amount`,
     description: `${log.name}: ${log.args.amount.toString()}\nToken: ${log.args.reserve}`,
     alertId: 'AE-AAVE-HIGH-TX-AMOUNT',
     severity: FindingSeverity.Medium,
     type: FindingType.Suspicious,
     everestId: AAVE_EVEREST_ID,
-    metadata: JSON.stringify(log),
+    metadata: {
+      txHash: txEvent.hash,
+    },
```

Comment:
The transaction hash is included in the Forta Explorer display, so this can probably be removed.
```suggestion
```

It also looks like the `description` field was used for data we would typically put in the `metadata` field.  In its current form, the `metadata` field would be unnecessary.  Perhaps the description field could be more general and the `metadata` field could contain the specifics?

Reply:
I updated the description to be more generic and included more details in the metadata field instead